src/engine/Actor.ts

```diff
@@ -289,14 +289,65 @@ module ex {
      * The unique identifier for the actor
      */
     public id: number = Actor.maxId++;
-    /** 
-     * The x coordinate of the actor (middle if anchor is (0.5, 0.5) left edge if anchor is (0, 0))
-     */ 
-    public x: number = 0;
-    /** 
-     * The y coordinate of the actor (middle if anchor is (0.5, 0.5) and top edge if anchor is (0, 0))
+        
+    /**
+     * The (x, y) position of the actor this will be in the middle of the actor if the [[anchor]] is set to (0.5, 0.5) which is default. If
+     * you want the (x, y) position to be the top left of the actor specify an anchor of (0, 0). 
+     */
+    public pos: Vector = new ex.Vector(0, 0);
+    
+    /**
+     * The position of the actor last frame (x, y) in pixels
+     */
+    public oldPos: Vector = new ex.Vector(0, 0);    
```

Comment:
How do people feel about "old"? What about "last" since it indicates last frame?